### DISCUSSION TITLE
fpfc desyncs the game

### DISCUSSION THREAD

**Discussion by TacticalGubbins** [kinsi55/CS_BeatSaber_Camera2#99] [Q&A] [answered]
Recently built a new PC and when this mod is installed and I am using fpfc the screen lags behind by about 2 seconds from whatever is happening in game. So if I move my mouse, it takes about 2 seconds to show on screen. when watching replays, the visuals are about 2 seconds behind the audio. The problem instantly goes away the second the mod is uninstalled. Works completely fine on the old PC and when in vr, but I haven't had any troubles with any other game on this build and would prefer to be able to use fpfc as needed. I'm using the default config and am running an i5 13600k and a rx6600 xt if that matters. Anyone know what might be causing this and how I can get the mod to work properly?

**Comment by kinsi55** [comment, marked as answer]
This is an issue with buggy AMD drivers, play around with the vsync setting for the game
